cities/admin.py

```diff
@@ -1,7 +1,8 @@
 from django.contrib import admin
+from django.contrib.gis import admin as geoadmin
 from .models import *
 
-class CitiesAdmin(admin.ModelAdmin):
+class CitiesAdmin(geoadmin.OSMGeoAdmin):#admin.ModelAdmin):
```

Comment:
Why the change here?

Reply:
This change should be propagated to make sure that we have an appropriate map in the Admin instead of Nasa's Map that has almost no data in it.

Reply:
@mhamzawey If you write a PR that adds a map to the different model admins I would review it and possibly merge it (if I have time).